core/src/test/java/ru/spbau/blackout/serializationutils/EfficientSerializableTest.java

```diff
@@ -0,0 +1,289 @@
+package ru.spbau.blackout.serializationutils;
+
+import com.badlogic.gdx.math.Vector2;
+
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.util.Random;
+
+import ru.spbau.blackout.utils.Utils;
+
+import static org.junit.Assert.*;
+
+public class EfficientSerializableTest {
+
+    @Test
+    public void serializeBooleanTest() throws Exception {
+        final int n = 10;
+        final Random random = new Random();
```

Comment:
One good practice is to set a fixed random seed, so that your tests become deterministic.

With deterministic tests it's much easier to detect when your (or other developers') future changes introduce  problems.